packages/venia-concept/src/ServiceWorker/__tests__/registerRoutes.spec.js

```diff
@@ -0,0 +1,123 @@
+import { THIRTY_DAYS, MAX_NUM_OF_IMAGES_TO_CACHE } from '../defaults';
+import { cacheHTMLPlugin } from '../Utilities/htmlHandler';
+import registerRoutes from '../registerRoutes';
+
+function StaleWhileRevalidate(options = {}) {
+    this.cacheName = options.cacheName;
+    this.plugins = options.plugins;
+}
+
+function CacheFirst(options = {}) {
+    this.cacheName = options.cacheName;
+    this.plugins = options.plugins;
+}
+
+function cacheableResponsePlugin() {}
+
+function expirationPlugin(options = {}) {
+    this.maxEntries = options.maxEntries;
+    this.maxAgeSeconds = options.maxAgeSeconds;
+}
+
+beforeAll(() => {
+    global.workbox = {
+        strategies: {
+            StaleWhileRevalidate,
+            CacheFirst
+        },
+        cacheableResponse: {
+            Plugin: cacheableResponsePlugin
+        },
+        expiration: {
+            Plugin: expirationPlugin
+        },
+        routing: {
+            registerRoute: function() {}
+        }
+    };
+});
+
+test("A total of 5 routes need to be registered using workbox's registerRoute API", () => {
```

Comment:
I think we can probably drop this test.

We're testing the five routes in more depth below, and any time we add a new route we'd have to come in and update this test. It just doesn't add much value.

Reply:
You are right but I added it to deliberately fail when a new route is added. @zetlen had a plan for a future feature that will auto-create these routes based on the app routes from `venia` in which case this test can be modified to suit that change and also fail if someone added a route signifying that they are changing and needs caution. But, I feel we should remove this test for not as it doesn't serve a lot of purpose.

Reply:
I meant this be a snapshot test equivalent.